webapp-django/crashstats/documentation/views.py

```diff
@@ -33,6 +38,29 @@ def home(request, default_context=None):
     return render(request, "documentation/home.html", context)
 
 
+@cache
+def read_whatsnew():
+    """Reads the WHATSNEW.rst file, parses it, and returns the HTML
+
+    :returns: HTML document as string
+
+    """
+    path = Path(settings.SOCORRO_ROOT) / "WHATSNEW.rst"
+
+    with open(path, "r") as fp:
+        data = fp.read()
+        parts = docutils.core.publish_parts(data, writer_name="html")
+
+    return parts["html_body"]
```

Comment:
I thought about doing this with the protected data access policy, too. Then it can also exist in three places (and be easier to maintain). I might do that another day.